.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,17 @@
+## Update Code
+- [x] #issue that needs to be picked up
+- [x] #PR that needs to be reviewed
```

Comment:
Where and what PR's  need to be reviewed? From my understanding, you have to review the data PR's first and then import the submodule with the most recent changes making sure to confirm you are on the latest commit. I think this should all be explicit in the template

Reply:
this was intended for the bugs that were not fixed from last time i.e. #352 and #360. I guess going forward we might not have them or they might be considered "enhancements". Going to remove it for now.

Reply:
I think the issue templates should be more generalized - is it cool if I work on this a little?